opencti-platform/opencti-front/src/private/components/common/lists/Filters.js

```diff
@@ -109,6 +109,7 @@ const Filters = ({
   } else {
     handleUpdateDateInputValues(usedFilters);
```

Comment:
Don't you need to change the condition in the  `handleUpdateDateInputValues` method because it will always be true as you give a default value to  `usedFilters = {}` ?
```suggestion
handleUpdateDateInputValues(usedFilters)
    if (filtersToApply !== {} ) {
```